lib/fizzy/value.hpp

```diff
@@ -13,8 +13,19 @@ union Value
     uint64_t i64;
 
     Value() = default;
-    constexpr Value(uint64_t v) noexcept : i64{v} {}
+    constexpr Value(unsigned int v) noexcept : i64{v} {}
```

Comment:
Maybe a comment why we need these `unsigned` constuctors

Reply:
Added.

Reply:
Instead of having 5 constructor for ints + deleted constructors, I can do all of that with a template constructor with some enable_if and some other magic. But I'm not sure we want that now. Maybe it's better to wait for floats before more cleanups.